src/main/java/logic/Ui.java

```diff
@@ -26,14 +26,20 @@ public void greet() {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         sb.append(logo);
-        sb.append("Hello! I'm logic.Duke\n");
+        sb.append("Hello! I'm Duke\n");
         sb.append("What can I do for you?");
         printStr(sb.toString());
     }
 
     //returns numbered list of tasks
-    public void printList(List<Task> taskList) {
-        StringBuilder sb = new StringBuilder("Here are the tasks in your list:\n");
+    public void printList(List<Task> taskList, boolean isMatching) {
```

Comment:
Is there a design reason to use `isMatching` here? `printList` seems to be a general method and should not know about matching tasks.